### ISSUE TITLE
New Edits B&A page

### ISSUE THREAD

**Issue by ah-bas-und-aer** [GuidaGG/basundaer#28]
General Bug: 
- bar in background color at the bottom of each page

Studio: (is not working recently)
- Hyperlink on Client Logos
- Instead of section "services“ set new section "references" wih two or three text coloumns
- move "services" below "coworking"
- Popover EN: "Overlay" instead of "Gesamt"

Projekte: 
- Add time period to the project info „year“ (2023–2024) or (since 2023)
- New filter option for sectors (economics, social, cultural ...) in a row below the old filters
- add another project info "With“ were we can put all out project partners
- background of a gallery is darker grey after the last images (at Ben's screen, I dont think it is a bug but its just loading (?))

Research: 
- Is it possible to add credits for simple image?
- Gallery: Row of Numbers on images is not fitting numbers in the credits
- Specific Gallery for logos with option to add hyperlinks (vlg. Flatterweg at bottom) logos could be smaller/ more in one row

**Comment by ah-bas-und-aer** [comment]
![screenshot-basundaer de-2024 10 17-10_25_42](https://github.com/user-attachments/assets/2a4497da-e303-4b42-b160-88582cad6c7d)

**Comment by ah-bas-und-aer** [comment]
> will there still be a services section + references section below coworking, or you mean just the new references section?

You did it just right, th way we see it now on the nomis page

**Comment by ah-bas-und-aer** [comment]
> _Gallery: Row of Numbers on images is not fitting numbers in the credits_'
> 
>     * the galleries are not showing in some projects, oder? there is a caption section but no images.

I dont understand, this refers only to the Research page, not Design, just to make this clear. 
F.I. in the post „ Research / Building Cultural Landscapes“ in the first gallery the images do have the numbers 1 and 2, 
in the second gallery it's 3 and 4 but in the caption it is 1 and 2. There should be written 3 and 4.

**Comment by ah-bas-und-aer** [comment]
> how is the priority of the project? In terms of timeline. Could we say we make a feedback meeting in the middle of november?

Yes, it is not urgent but nice to have till mid of december. Yes, we can have a meeting. F.I. next week?

**Comment by GuidaGG** [comment]
I will (hopefully) finish the Design page next monday.
Then the research page, i will try to find a date to work and by then you can add the rest of the content in studio. :)

**Comment by GuidaGG** [comment]
- Design Page:
 - simplify image code
 - add 2 words for all (all sectors, all categores)
 
- Research
- use studio logo gallery as  reference for logo
- add more spacing on the bottom

- Contact
- less spacing between words

-
